staging/src/k8s.io/cloud-provider/controllers/service/controller_test.go

```diff
@@ -68,6 +68,7 @@ func newService(name string, serviceType v1.ServiceType, tweaks ...serviceTweak)
 		},
 		Spec: v1.ServiceSpec{
 			Type: serviceType,
+			Ports: makeServicePort(v1.ProtocolTCP, 0),
```

Comment:
0 is not a valid port number - better to set it to 80 or something

Reply:
0 is the symbol indicating whether the service target port is set or not.

```
func makeServicePort(protocol v1.Protocol, targetPort int) []v1.ServicePort {
	sp := v1.ServicePort{Port: 80, Protocol: protocol}
	if targetPort > 0 {
		sp.TargetPort = intstr.FromInt(targetPort)
	}
	return []v1.ServicePort{sp}
}
```

Reply:
Ahh, this was unclear.  I assumed it was the port number.